hack/build/ci/update-renovate-json5.py

```diff
@@ -0,0 +1,24 @@
+import json5
+import sys
+
+# version file contains a list of strings
+versionFile = sys.argv[1]
+renovateFile = sys.argv[2]
+
+# read versions to list
+versions = ['$default']
```

Comment:
it's not a blocker, but if you can install `pip install black` and format file, it would be super nice :)